packages/embark-transaction-manager/src/lib/eth_signTypedData.ts

```diff
@@ -0,0 +1,45 @@
+import { sign, transaction } from "@omisego/omg-js-util";
+import { Callback, Embark, Logger } /* supplied by @types/embark in packages/embark-typings */ from "embark";
+import { __ } from "embark-i18n";
+import Web3 from "web3";
+const { blockchain: blockchainConstants } = require("embark-core/constants");
+
+export default class EthSignTypedData {
```

Comment:
Because geth and most other nodes don't support it (yet or ever). Instead they return the error "The method eth_signTypedData_v3 does not exist/is not available".